cmake/create_test.cmake

```diff
@@ -17,6 +17,26 @@ function(ginkgo_create_test test_name)
     add_test(NAME ${REL_BINARY_DIR}/${test_name} COMMAND ${TEST_TARGET_NAME})
 endfunction(ginkgo_create_test)
 
+function(ginkgo_create_dpcpp_test test_name)
+    file(RELATIVE_PATH REL_BINARY_DIR
+        ${PROJECT_BINARY_DIR} ${CMAKE_CURRENT_BINARY_DIR})
+    string(REPLACE "/" "_" TEST_TARGET_NAME "${REL_BINARY_DIR}/${test_name}")
+    add_executable(${TEST_TARGET_NAME} ${test_name}.dp.cpp)
+    target_compile_features("${TEST_TARGET_NAME}" PUBLIC cxx_std_17)
```

Comment:
In the dpct beta update 10, they describe something related to std17 and `std::max_element`.
Maybe dpct does not require std17 anymore.
Because we do not use dpct header currently, could you try std14?

Note. I saw the DPCT beta update 10 release note yesterday, but I can not find it anymore.